examples/docs_12_quadratic_symmetric.jl

```diff
@@ -194,8 +194,68 @@ function build_reynolds_unique(p::Array{T, N}) where {T <: Number, N}
     end
 end
 
-lmo_unique = FrankWolfe.SymmetricLMO(lmo_naive, build_reynolds_unique(p), reynolds_adjoint)
+lmo_unique = FrankWolfe.SymmetricLMO(lmo_naive, build_reynolds_unique(p))
+@time FrankWolfe.blended_pairwise_conditional_gradient(f, grad!, lmo_unique, FrankWolfe.ActiveSetQuadratic([(one(T), x0)], LinearAlgebra.I, -p); verbose, lazy=true, line_search=FrankWolfe.Shortstep(one(T)), max_iteration=10) #hide
 asq_unique = FrankWolfe.ActiveSetQuadratic([(one(T), x0)], LinearAlgebra.I, -p)
-FrankWolfe.blended_pairwise_conditional_gradient(f, grad!, lmo_unique, asq_unique; verbose=false, lazy=true, line_search=FrankWolfe.Shortstep(one(T)), max_iteration=100) #hide
-@time FrankWolfe.blended_pairwise_conditional_gradient(f, grad!, lmo_unique, asq_unique; verbose, lazy=true, line_search=FrankWolfe.Shortstep(one(T)))
+@time FrankWolfe.blended_pairwise_conditional_gradient(f, grad!, lmo_unique, asq_unique; verbose, lazy=true, line_search=FrankWolfe.Shortstep(one(T)), max_iteration)
+println() #hide
+
+# ### Physical reduction of the iterate
```

Comment:
physical is maybe not the obvious choice of words, "Reduction of the iterate memory footprint"?

Reply:
Storage cost reduction?

Reply:
Memory footprint sounds good, I'll edit this, thanks.